dataprofiler/labelers/column_name_model.py

```diff
@@ -39,7 +39,16 @@ def __init__(self, parameters=None):
         parameters.setdefault("include_label", True)
         parameters.setdefault("negative_threshold_config", None)
 
-        # initialize class
+        label_mapping = {"not": "implemented"}
+        self.set_label_mapping(label_mapping)
+        logger.info(
+            "For MVP implementation, the `ColumnNameModel`"
+            "does not implement the `label_mapping` utility."
+            "'prediction -> label' mapping is handled in the"
+            "post-processor using the data index / row value."
+        )
```

Comment:
RE: comment above... adding a logger.info so the user is aware of the label_mapping situation for MVP implementation